Fixing EQ7

tdegeorge opened the pull request:
Closes #562

tdegeorge commented:
I changed the majority of this section based on @AbbyANoble 's feedback. Please double the graphs as I struggled with some of them! :)

siwelwerd commented on source/precalculus/source/01-EQ/07.ptx:
```diff
@@ -159,227 +393,923 @@
-          p += text("]", (-12,0), color="#0088ff", fontsize=18)
-          p += text("(", (-5,0), color="#0088ff", fontsize=18)
+          #p += TBILPrecal.inequality_plot(start=-2,end=7,label_endpoints=False)
+          p = arrow((-10,0),(12,0),color="black", width=1, arrowsize=1, aspect_ratio=1,head=2)
```
```suggestion
          p = arrow((-10,0),(10,0),color="black", width=1, arrowsize=1, aspect_ratio=1,head=2)
```

siwelwerd commented on source/precalculus/source/01-EQ/07.ptx:
```diff
@@ -159,227 +393,923 @@
+          for i in range(-9,10):
+            p += line([(i,-0.2),(i,0.2)],color="black")
+            p += text(f"${i}$", (i,-0.6),color="black")
+          p += line([(-10,0),(2,0)],color="#0088ff", thickness=3, aspect_ratio=1)
+          p += line([(4,0),(10,0)],color="#0088ff", thickness=3, aspect_ratio=1)
```
```suggestion
          p += arrow((2,0),(-10,0),color="#0088ff", thickness=3, arrowsize=3)
          p += arrow((4,0),(10,0),color="#0088ff", thickness=3, arrowsize=3)
```
I think it looks better if the shaded part are also arrows

siwelwerd commented on source/precalculus/source/01-EQ/07.ptx:
```diff
@@ -159,227 +393,923 @@
+          <!--<xi:include parse="text" href="../../sage/common.sage"/> -->
+          #p = TBILPrecal.numberline_plot()
+          #p += TBILPrecal.inequality_plot(start=-2,end=7,label_endpoints=False)
+          p = arrow((-10,0),(12,0),color="black", width=1, arrowsize=1, aspect_ratio=1,head=2)
```
```suggestion
          p = arrow((-10,0),(10,0),color="black", width=1, arrowsize=1, aspect_ratio=1,head=2)
```

siwelwerd commented on source/precalculus/source/01-EQ/07.ptx:
```diff
@@ -159,227 +393,923 @@
+          for i in range(-9,10):
+            p += line([(i,-0.2),(i,0.2)],color="black")
+            p += text(f"${i}$", (i,-0.6),color="black")
+          p += line([(-10,0),(-2,0)],color="#0088ff", thickness=3, aspect_ratio=1)
```
```suggestion
          p += arrow((-2,0),(-10,0),color="#0088ff", thickness=3, arrowsize=3)
```

AbbyANoble commented on source/precalculus/source/01-EQ/07.ptx:
```diff
@@ -13,93 +13,395 @@
+  <task>
+    <statement>
+      <p>
+        Use a graphing utility to graph the function <m>f(x)=x^2-6x+5</m>. What is the value of <m>f(0)</m>?
```
This is before the functions chapter, so we probably don't need to use function notation.

AbbyANoble replied:
Coming back to this, maybe we should move this section? If we make it PR7 instead, then they would do it after they have solved the corresponding equations, learned about functions in general, learned more specific stuff about polynomial and rational functions, and graphed them. Or maybe this makes sense as two sections? Polynomial inequalities then Rational inequalities?

AbbyANoble commented on source/precalculus/source/01-EQ/07.ptx:
```diff
@@ -13,93 +13,395 @@
+
+  <introduction>
+    <p>
+      Consider the quadratic inequality <me> x^2-6x+5 \gt 0.</me>
```
To switch to giving them a graph:
```suggestion
      Consider the graph shown below.
       <figure width="100%" xml:id="EQ7-quadratic">
          <image xml:id="EQ7-quadratic-image">
            <description>Graph of a quadratic</description>
            <sageplot>
              f(x) = (x^2-6*x+5)
                p=plot(f, (x, -1, 5), ymin=-7, ymax=10, color='blue', thickness=3)
            </sageplot>
          </image>
        </figure>
```

tdegeorge replied:
So, if we are moving this section, do we want students to graph it or provide the graph for them?

AbbyANoble replied:
I'd still provide it to cut out some of the time that would be spent graphing it? Then we can all look at the same thing. (I'm finding that I can't get through many of our sections in a 75 minute class period!)

siwelwerd commented on source/precalculus/source/01-EQ/07.ptx:
```diff
@@ -13,93 +13,395 @@
+  <definition xml:id="def-quadratic-inequality">
+    <statement>
+      <p>
+        A <term>quadratic inequality</term> is a mathematical statement that relates a quadratic expression (using <m>\lt, \le, \gt, \ge</m>) to another.
```
Yeah, do we want to say $x^2 \leq x^2$ is a quadratic inequality?  Tonya's definition says yes, but Abby's says no.

AbbyANoble replied:
@siwelwerd - Does the pretext here look right for my list? I did it in the editor rather than firing up a codespace.

siwelwerd replied:
Yep, if @tdegeorge accepts this we can double check the preview.

AbbyANoble commented on source/precalculus/source/01-EQ/07.ptx:
```diff
@@ -13,93 +13,395 @@
+
+  <remark>
+    <p>
+      From <xref ref="graphs-of-quadratic-inequalities"/>, we saw that a function could have <m>y</m>-values that are positive, negative, or zero, which can then help us find values of <m>x</m> to solve inequalities. Let's now look at how we can solve inequalities using algebra.
```
```suggestion
      From <xref ref="graphs-of-quadratic-inequalities"/>, we saw that a graph could have <m>y</m>-values that are positive, negative, or zero, which can then help us find values of <m>x</m> to solve inequalities. Let's now look at how we can solve inequalities using algebra.
```

AbbyANoble commented on source/precalculus/source/01-EQ/07.ptx:
```diff
@@ -13,93 +13,395 @@
+      </statement>
+      <answer>
+        <p>
+          A and D
```
```suggestion
          A and C
```

AbbyANoble commented on source/precalculus/source/01-EQ/07.ptx:
```diff
@@ -13,93 +13,395 @@
     <introduction>
-      <p>Consider the quadratic inequality <me>x^2-4x-32\gt0</me>.</p>
+      <p>
+        Let's look at how we can determine the solutions to a quadratic inequality using a number line. Consider the quadratic inequality <me>x^2-4x-32\ge 0</me>.
```
```suggestion
        Let's look at how we can algebraically determine the solutions to a quadratic inequality using a number line. Consider the quadratic inequality <me>x^2-4x-32\ge 0</me>.
```

AbbyANoble commented on source/precalculus/source/01-EQ/07.ptx:
```diff
@@ -13,93 +13,395 @@
       <statement>
-        <p>Use a graphing utility to graph the function <m>f(x)=x^2-4x-32</m>.  Which part of the graph represents where <m>x^2-4x-32\gt0</m>?</p>
+        <p>
+          What is the factored form of the quadratic (left-hand side of the inequality)?
+          <ol marker="A." cols="2">
+            <li><m>(x-16)(x+2)</m></li>
+            <li><m>(x-4)(x+8)</m></li>
+            <li><m>(x+4)(x-8)</m></li>
+            <li><m>(x-4)(x-8)</m></li>
+          </ol>
```
I think we could probably combine this and the next part into one since they should know how to solve quadratic equations. Maybe something like "We'll start by solving the corresponding quadratic _equation_ x^2-4x-32=0. What are the solution(s)?

tdegeorge commented:
@siwelwerd  - I believe I addressed all the comments/suggestions made for this section. Just double check the images in some of the answers (like the number lines). In codechat, the images would not show, so I am not entirely sure they are working/correct. Other than that, this should be ready to split into the two sections we talked about earlier.

siwelwerd commented:
The graphs look good to me. I went ahead and updated the number lines to use our new library functions, it's way cleaner to read in the code now.
